### PR TITLE
2066 review resources

### LINKED ISSUE #2066: [2.3] review Resources concept
Command `juju charm resources` becomes `juju charm-resources`.
See PR https://github.com/juju/juju/pull/7695

Command `juju attach` becomes `juju attach-resource`.
See PR https://github.com/juju/juju/pull/7693

### PR DESCRIPTION
**Pull request by pmatulis**
Fixes #2066

### CODE REVIEW

File: src/en/charms-resources.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+Title: Resources
+TODO:  Critical: Improve examples
+
+# Juju Resources
+
+A Juju *resource* is additional content/files that a charm needs in order to
```
Comment (evilnick) [inline src/en/charms-resources.md]: it doesn't *always* need them - they may be optional. Perhaps "that a charm can make use of or may require"

File: src/en/charms-resources.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+run.
+
+There are three ways to examine resources:
```
Comment (evilnick) [inline src/en/charms-resources.md]: three types of resources?

File: src/en/charms-resources.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Resource  Revision
+wal-e     0
+```
+
+## Application resources
+
+These are resources that a deployed application is currently using across all
+its units and are discoverable with the `resources` command. In contrast to the
+`charm-resources` command, this command needs a controller (that it queries) as
+well as deployed charm.
```
Comment (evilnick) [inline src/en/charms-resources.md]: as a deployed charm

File: src/en/developer-resources.md [hunk 4]
```diff
@@ -1,25 +1,24 @@
-Title: Writing charms using resources
+Title: Resources | Developer guide
 
-# Writing charms that use resources
+# Developer resources
```
Comment (evilnick) [inline src/en/developer-resources.md]: They aren't really 'developer resources'. Perhaps 'Using resources' would be better

File: src/en/charms-resources.md (hunk 1 above)
Comment (pmatulis) [reply src/en/charms-resources.md]: OK

File: src/en/charms-resources.md (hunk 2 above)
Comment (pmatulis) [reply src/en/charms-resources.md]: Not so simple. *Charm resources* are simply what resources a charm is defined with. Once the charm's application is running, you can look at the other two types, in a runtime fashion, from either a application or a unit standpoint. So they all seem intrinsically equivalent to me, hence "examining".

File: src/en/charms-resources.md (hunk 3 above)
Comment (pmatulis) [reply src/en/charms-resources.md]: OK

File: src/en/developer-resources.md (hunk 4 above)
Comment (pmatulis) [reply src/en/developer-resources.md]: OK

Comment (pmatulis) [reply src/en/developer-resources.md]: Should I leave the menu alone (*Juju Resources*)?

File: src/en/charms-resources.md (hunk 2 above)
Comment (evilnick) [reply src/en/charms-resources.md]: ok

File: src/en/developer-resources.md (hunk 4 above)
Comment (evilnick) [reply src/en/developer-resources.md]: the menu item is ok

**Comment by evilnick** [comment]
thanks :cake:

**Pull request merged**